apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.ts

```diff
@@ -29,4 +32,41 @@ router.get(
   }),
 );
 
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    if (!res.locals.authz_data.has_course_permission_edit) {
+      throw new error.HttpStatusError(403, 'Access denied (must be course editor)');
+    }
+    if (req.body.__action === 'delete_drafts') {
+      const questions = await queryRows(
+        sql.select_drafts_by_course_id,
+        { course_id: res.locals.course.id.toString() },
+        QuestionSchema,
```

Comment:
could send all questions to avoid multiple queries, but want to avoid sending too much data in trpc